src/commands/schema/push.ts

```diff
@@ -10,9 +10,9 @@ export default class PushSchemaCommand extends SchemaCommand {
       description: "Push the change without a diff or schema version check",
       default: false,
     }),
-    staged: Flags.boolean({
+    unstaged: Flags.boolean({
       description:
-        "Stages the schema change, instead of applying it immediately",
+        "Skips the staged schema. This will cause indexes to be temporarily unavailable.",
```

Comment:
how about something with more emphasis on the verb and outcome? e.g.,: `Skip staging the schema and make the schema active immediately. This will cause indexes to be temporarily unavailable.`

Reply:
Good idea, I'll update.